.github/codecov.yml

```diff
@@ -8,6 +8,14 @@ coverage:
   range: "40...70"
   status:
     patch: off
+  notify:
+    slack:
+      default:
+        url: secret:v1::tXC7VwEIKYjNU8HRgRv2GdKOSCt5UzpykKZb+o1eCDqBgb2PEqwE3A26QUPYMLo4BO2qtrJhFIvwhUvlPwyzDCNGoNiuZfXr0UeZZ0y1TcZu672R/NBNMwEPO/e1Ye0pHxjzKHnuH7HqbjFucox/RBQLtiL3J56SWGE3JtbkC6o=
```

Comment:
@jsjoeio Is this a secret credential? Can it be made public?

Reply:
Yes and yes! @bryphe-coder showed me how to do this. This is the recommended approach from Codecov

![image](https://user-images.githubusercontent.com/3806031/142690512-fd91847f-a4d1-4875-bce0-a11dd4501be2.png)